services/example/pkg/resource/book.go

```diff
@@ -0,0 +1,19 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package resource
+
+// bookResource implements the `aws-service-operator-k8s/pkg/types.AWSResource`
+// interface
+type bookResource struct {
+}
```

Comment:
Note for reviewers: eventually this object will *wrap* the runtime.Object type that matches it (in the case here, that would be `services/example/apis/v1alpha1/book.go:Book`) and add some other metadata about relationships between the object and other objects.